PyPDF2/_writer.py

```diff
@@ -794,6 +812,25 @@ def write(self, stream: StreamType) -> None:
         self._write_trailer(stream)
         stream.write(b_(f"\nstartxref\n{xref_location}\n%%EOF\n"))  # eof
 
+    def write(self, fileobj: StrByteType) -> None:
```

Comment:
```suggestion
    def write(self, stream: StrByteType) -> None:
```

Pleas don't change the name. If we change it, we need to deprecate the old name in order not to break other peoples workflow. People could use `writer.write(stream=something)`. That would break if we rename it to `fileobj`.